cmd/up/exec_windows.go

```diff
@@ -5,24 +5,21 @@ package up
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"os/exec"
-	"runtime"
+	"path/filepath"
 	"strings"
 	"syscall"
 )
 
+// GetCommandToExec implementation for windows GOOS
 func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*exec.Cmd, error) {
-	var c *exec.Cmd
-	if runtime.GOOS != "windows" {
-		c = exec.Command("bash", "-c", strings.Join(cmd, " "))
-	} else {
-		binary, err := expandExecutableInCurrentDirectory(cmd[0], he.workdir)
-		if err != nil {
-			return nil, err
-		}
-		c = exec.Command(binary, cmd[1:]...)
+	binary, err := expandExecutableInCurrentDirectory(cmd[0], he.workdir)
+	if err != nil {
+		return nil, err
 	}
+	c := exec.Command(binary, cmd[1:]...)
```

Comment:
What if the command is just the binary? I think `cmd[1:]...` will be out of index and return a runtime error